### ISSUE TITLE
Create custom splash screens

### ISSUE THREAD

**Issue by NeoTheThird** [ubports/ubuntu-touch#240]
Some devices are still using the Android-bootscreen. That should be changed to something like this:

![fairphone](https://user-images.githubusercontent.com/12068939/30335409-926c235c-97e2-11e7-85b5-b2d15dbc73ab.jpeg)

The following devices are still missing:

- [x]  Fairphone 2
- [ ]  Nexus 5
- [ ]  Nexus 4
- [ ]  Nexus 7
- [x]  Oneplus 1

**Comment by nfsprodriver** [comment]
Just created a simple ubports bootscrenn for the Nexus 5. Can you give me link the the upper one so I can edit it for hammerhead?

**Comment by nfsprodriver** [comment]
![boot](https://user-images.githubusercontent.com/6537842/30409451-5a5bc868-9905-11e7-8fba-db4a7e822214.jpg)

**Comment by mymike00** [comment]
If you need, I can help making some images...

**Comment by Flohack74** [comment]
Info: The Fairphone 2 will get the new bootsplash with OTA-2. All other phones will need to have 2 updates because of some partition trick to place the new screen. That means, users will update to OTA-2, then to OTA-3. Eventually people who skip OTA-2 will get this done in OTA-4 and so on. Ticket therefore stays open until we get positive confirmation that it worked for the majority of users.

**Comment by NeoTheThird** [comment]
@nfsprodriver That looks really cool, good work!

@mymike00 Thanks, we'll need to talk some more before ota 3 when, as flo explained, the feature will land for most devices.

**Comment by mymike00** [comment]
ok @NeoTheThird, I aksed if I could help so OTA2 could be released faster. now the feature is in OTA 3 there's a lot of time to do this. anyway, if you need help, just ping me ;)

**Comment by NeoTheThird** [comment]
@mymike00 will do, thank you!

**Comment by ChloeWolfieGirl** [comment]
@NeoTheThird You put OnePlus 2, did you mean that, or did you mean OnePlus One?

**Comment by NeoTheThird** [comment]
@ChloeWolfieGirl Ooops, thanks for noticing. Fixed.

**Comment by mymike00** [comment]
where can I get the original images?
and should I follow some rules to make the new ones? like "powered by ubuntu touch" under the brand logo and "supported by: ubports" at the bottom?

**Comment by mymike00** [comment]
do you still need help in creating the images?

**Comment by ebeton-ro** [comment]
@nfsprodriver that nexus5 splash screen it is awesome

**Comment by nfsprodriver** [comment]
Thanks! Also created some for N4, N7. But I think I forgot to save them :(

**Comment by nfsprodriver** [comment]
We just may replace the 5 by 4 and 7 or replace whole Nexus logo. It's not much work.

**Comment by ChloeWolfieGirl** [comment]
The opo has a tick next to it but on rc, it still doesn't have the new boot screen

**Comment by NeoTheThird** [comment]
We'll have to take a different approach with this one. Instead of shipping it in the image, we'll make the Installer flash the bootsplash partition. If someone already wants to try this for the Fairphone 2, download [this](https://github.com/ubports/device-files/blob/ubp-5.1/FP2/partitions/splash.bin) and flash the partition with `fastboot flash splash splash.bin`. **DO NOT TRY THIS ON A DEVICE OTHER THAN FP2**.

**Comment by NeoTheThird** [comment]
Just waiting on some architectural changes in the installer and i'll get to work on this one.

**Comment by dobey** [comment]
If we're going to do this, can we at least make them NOT be white background? Maybe the SuruDark theme colors instead? :)

That's one thing I hate on a phone, especially when I have to reboot at night. Full bright screen with 80%+ pixels being white, is bad for the eyes.

**Comment by mariogrip** [comment]
I do agree with @dobey there, My eyes dies a bit every time i have to reboot at night...

**Comment by PhoenixLandPirate** [comment]
I also support @dobey 's message, I don't know how the logo will look on a dark/black background, but the current white, is just asking for pain during the night.

**Comment by Nikfrager** [comment]
Please don't forget the Moto G2!

**Comment by Flohack74** [comment]
Probably we need to change all of them now. The logo needs to be modified...

**Comment by Flohack74** [comment]
The problem is that now after almost 1 year Canonical does not want the word Touch so close to Ubuntu ;) - stand by, updates follow. We will have to do some final adjustments in our strategy...

**Comment by dobey** [comment]
Also, please include the source images, rather than just the built images one flashes to the device. We need sources (preferably SVG) in case modifications need to be made later, and so we can generate images for new devices that get ported. Binary-only isn't acceptable.

**Comment by Flohack74** [comment]
I put an issue for that here: https://github.com/rubencarneiro/custom_bootlogos/issues/5

**Comment by mateosalta** [comment]
Here is a bin for the opo that removes the "powered by android" text, note that the opo is very picky about the files for it's boot and if it goes wrong this hard bricks the device... (recoverable but very difficult) Because of this I had to directly edit the bin, so there are no source files. The logo builders for this device you find online are for older versions of android and will brick the device.

https://t.me/UBports_UX/1035

**Comment by Flohack74** [comment]
I am closing this here, Ruben has already done screens for all devices. Sorry for not seeing this earlier!

**Comment by Flohack74** [comment]
See https://gitlab.com/rubencarneiro/custom_bootlogos

**Comment by Nikfrager** [comment]
For all devices? But a screen for the Moto G2 is still missing. Or is one of the others suitable?

**Comment by Flohack74** [comment]
Moto G2 is not an official core device. In this case we will not overwrite the default one.

**Comment by mateosalta** [comment]
i see that ruben's link is down, can we these posted somewhere if we want the sources to make/modify say if we want a dark theme version - also was the opo boot screen shipping somewhere?

**Comment by mateosalta** [comment]
how was the opo screen made? which android version was it targeted at, the opo will brick if there is a mismatch

**Comment by lduboeuf** [comment]
sorry maybe something i've made wrongly but still see Google's logo in black background  on Nexus 5 at boot after a fresh install.

**Comment by Flohack74** [comment]
@mateosalta I flashed the latest one from Ruben and it worked perfectly. Did not even know I could brick it but it was OK ;)

**Comment by lduboeuf** [comment]
can someone confirm ubports splashscreens are shipped with images ? I don't have any for now on Nexus 5 (installed via ubports-installer 16.04 OTA6)?

**Comment by Flohack74** [comment]
Sorry but we did not implement it in all devices. Lets target this again for OTA-7. I will have to ask the team if we reopen this issue here.

**Comment by jonnius** [comment]
It seems Ruben has deleted his repository. Does anybody know why or if the boot logos are still somewhere arround? I've got an outdated fork, containing at least the white boot screens: https://github.com/jonnius/custom_bootlogos

**Comment by Flohack74** [comment]
he is on Gitlab now: https://gitlab.com/rubencarneiro/custom_bootlogos

**Comment by hummlbach** [comment]
![screenshot20190122_232117717](https://user-images.githubusercontent.com/10819327/51569398-8e248400-1e9c-11e9-92ba-b2bcbbe0057a.png)
:(

**Comment by Flohack74** [comment]
![grafik](https://user-images.githubusercontent.com/16186470/52507535-1bccd700-2bf2-11e9-9a4a-330b47fdaee1.png)
Fix your device!

**Comment by mateosalta** [comment]
he must have the repo not public then...

**Comment by rubencarneiro** [comment]
So for all that want the bootlogos is available at https://rubencarneiro.github.io/rubencarneiro.io//Ubuntu-Touch-Device-Bootlogos.markdown/
